modules/utility/init.zsh

```diff
@@ -6,7 +6,8 @@
 # Colours
 #
 
-if (( ${terminfo[colors]} >= 8 )); then
```

Comment:
In that case, perhaps:

 `if [[ ! -z ${terminfo[colors]} ]] && (( ${terminfo[colors]} >= 8 )); then`

I realize that the `! -z` is a double negative in reality, but I think it makes it more clear *why* we're checking it.

Doing that, it should first ensure that the key-value pair exists, and if it does, then verify that colors >= 8.